OpenRA.Mods.Cnc/Traits/Mine.cs

```diff
@@ -58,6 +58,12 @@ bool ICrushable.CrushableBy(Actor self, Actor crusher, BitSet<CrushClass> crushC
 
 			return info.CrushClasses.Overlaps(crushClasses);
 		}
+
+		bool ICrushable.IsCrushable(Actor self, BitSet<CrushClass> crushClasses, out LongBitSet<PlayerBitMask> blocking)
+		{
+			blocking = default(LongBitSet<PlayerBitMask>);
+			return false;
```

Comment:
Can you explain this?
Without any comments, my intuition is that setting `blocking` to the default (nothing is blocked) means that it is crushable, but `return false` means that its not crushable!

Reply:
Here we fallback to the old code instead, because there are cases in the crushing logic for mines that we cant handle in the cache, for example if an actor has immunity to the mine etc.

Reply:
Ah, I see. So the bool is actually a "handled by the new logic" flag?

What do you think about changing the method signature to something like `LongBitSet<PlayerBitMask> ICrushable.BlocksMovementFor(Actor self, out bool fallbackToSlowBlockingCheck)`? I'm not convinced on my exact naming suggestion, but something along these lines would make it much clearer.

Reply:
Or, perhaps better,
`bool TryCalculateOwnerBlocking(Actor self, BitSet<CrushClass> crushClasses, out LongBitSet<PlayerBitMask> blocking)`?